frontend/public/components/nav/section.tsx

```diff
@@ -21,6 +21,53 @@ const navSectionStateToProps = (state: RootState, {required}) => {
   };
 };
 
+const mapChild = (c, activeChild, flags, activeNamespace) => {
```

Comment:
Better to add types to function declarations

Reply:
Types added.

I considered the function to be single-purposed, just extracted from former render() closure to improve readability.

As requested, I changed it to access `this` directly but moved inside class definition as it expects `this` to meet certain contract.